commit 31c93397bde772764cda3058e16f9cef61895090
Author: Joshua Nelson <jyn514@gmail.com>
Date:   Mon Feb 8 22:51:21 2021 -0500

    Use format string in bootstrap panic instead of a string directly
    
    This fixes the following warning when compiling with nightly:
    
    ```
    warning: panic message is not a string literal
        --> src/bootstrap/builder.rs:1515:24
         |
    1515 |                 panic!(out);
         |                        ^^^
         |
         = note: `#[warn(non_fmt_panic)]` on by default
         = note: this is no longer accepted in Rust 2021
    help: add a "{}" format string to Display the message
         |
    1515 |                 panic!("{}", out);
         |                        ^^^^^
    help: or use std::panic::panic_any instead
         |
    1515 |                 std::panic::panic_any(out);
         |                 ^^^^^^^^^^^^^^^^^^^^^^
    ```

diff --git a/src/bootstrap/builder.rs b/src/bootstrap/builder.rs
index f1a160250db..0f5fcb4af40 100644
--- a/src/bootstrap/builder.rs
+++ b/src/bootstrap/builder.rs
@@ -1490,7 +1490,7 @@ pub fn ensure<S: Step>(&'a self, step: S) -> S::Output {
                 for el in stack.iter().rev() {
                     out += &format!("\t{:?}\n", el);
                 }
-                panic!(out);
+                panic!("{}", out);
             }
             if let Some(out) = self.cache.get(&step) {
                 self.verbose(&format!("{}c {:?}", "  ".repeat(stack.len()), step));
